### ISSUE TITLE
Facilitate angle-limited integrations for SESANS

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#2282]
User RobD @ ISIS notes that the SESANS codebase is written to accept non-infinite angular integration limits, but that there is presently no way to provide such limits from the GUI, only from scripting. This is not a User-friendly state of affairs.

**Describe the solution you'd like**
There should be a way to provide angular limits for SESANS analysis in the GUI. Rob suggests adding something to the resolution smearing tab?

**Comment by wimbouwman** [comment]
I agree with RobD. I confirm also that it is best to take the angular limit and not a Q-limit for the TOF SESANS. The angular limits are the most general way to describe the integration limits.
